src/config/logger.ts

```diff
@@ -117,12 +117,7 @@ export const getUnsafeLogger = (name: string, fields?: Record<string, unknown>):
 	return logger.child({ ...fields });
 };
 
-
-export const cloneAllowedLogFields = (fields: Record<string, unknown>) => {
-	const allowedFields = { ...fields };
-	delete allowedFields.name;
-	return allowedFields;
-};
+export const cloneAllowedLogFields = (fields: Record<string, unknown>) => omit(fields, ["name"]);
```

Comment:
havn't used `omit` before, looks rather handy!

Reply:
sure is! lodash is full of goodies like that!